static/docs/commands-reference/get.md

```diff
@@ -44,4 +44,31 @@ created in the current working directory with its original file name.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-<!--  ## Example -->
+## Example
+
+Note that `dvc get` can be used form anywhere in the file system, as long as DVC
+is [installed](/doc/get-started/install). For this example we assume an empty
+current working directory.
+
+We can use `dvc get` to download the resulting model file from our
+[get started example](https://github.com/iterative/example-get-started), which
+is a DVC project external to the directory we're in). The desired file is
+located in the `model.pkl` path.
+
+```dvc
+$ dvc get https://github.com/iterative/example-get-started model.pkl
+Preparing to download data from 'https://remote.dvc.org/get-started'
+...
+$ ls
+model.pkl
+```
+
+Note that the `model.pkl` file doesn't actually exist in the
+[data directory](https://github.com/iterative/example-get-started/tree/master/)
+of the external Git repo. Instead, the corresponding DVC-file
+[train.dvc](https://github.com/iterative/example-get-started/blob/master/train.dvc)
+is found, which specifies `model.pkl` in its outputs (`outs`). DVC then
+reproduces this stage locally by [pulling](/doc/commands-reference/pull) the
```

Comment:
no need to use "reproduce" term here. I don't think it's that relevant here. May be just "DVC then pulls ..."

Reply:
Also fixed in d20b190